.tmuxinator.yml

```diff
@@ -21,8 +21,14 @@ windows:
             - source scripts/tmux-user-shell.sh
   - federation:
       panes:
-        - federation:
-          - tail -n +0 -F $FM_LOGS_DIR/fedimint-*.log
```

Comment:
`$FM_LOGS_DIR/fedimint-*.log` was slightly wrong. Logs are in `$FM_LOGS_DIR/fedimintd-*.log`. Missing the `d` in `fedimintd`.